bin/jurassic-tube-setup.sh

```diff
@@ -43,8 +43,8 @@ echo
 ${PWD}/docker/bin/jt/config.sh username ${username}
 ${PWD}/docker/bin/jt/config.sh subdomain ${subdomain}
 
-echo "Setup complete!" 
-echo "Use the command: docker/bin/jt/tunnel.sh from the root directory of your WC Payments project to start running Jurassic Tube."
+echo "Setup complete!"
+echo "Use the command: npm run start:jurassic-tube from the root directory of your WC Payments project to start running Jurassic Tube."
```

Comment:
This needs to change in respect of https://github.com/Automattic/woocommerce-payments/pull/1578/files#r614590845.

Reply:
Made changes in commit https://github.com/Automattic/woocommerce-payments/pull/1578/commits/23abb577033b97e8824c195817fdf8a980dfa2f0